main.go

```diff
@@ -231,7 +233,7 @@ func setupControllers(mgr ctrl.Manager, sw *watch.ControllerSwitch, tracker *rea
 	<-setupFinished
 
 	// initialize OPA
-	driver := local.New(local.Tracing(false))
+	driver := local.New(local.Tracing(false), local.DisableBuiltins(disabledBuiltin.ToSlice()...))
```

Comment:
Do we need to worry about duplicate strings in the slice, or is that handled elsewhere?